geonode/qgis_server/views.py

```diff
@@ -810,6 +811,21 @@ def default_qml_style(request, layername, style_name=None):
         qgis_layer.default_style = style
         qgis_layer.save()
 
+        # Recreate thumbnail
```

Comment:
Can you refactor thumbnail default creation to inside of `create_qgis_server_thumbnail`, so this view function can just call one function?

Also please include in unittest of setting default_qml_style that the thumbnail is indeed recreated.